virttest/qemu_monitor.py

```diff
@@ -2210,3 +2210,15 @@ def system_powerdown(self):
         if not utils_misc.wait_for(lambda: self.get_event(qmp_event), 120):
             raise QMPEventError(cmd, qmp_event, self.name)
         logging.info("%s QMP event received" % qmp_event)
+
+    def transaction(self, args):
+        """
+        Atomically operate on a group of one or more block devices.
+
+        :param args: information needed for the device snapshot.
+
+        :return: nothing on success
+                 If @device is not a valid block device, DeviceNotFound
```

Comment:
This is picked up from source code of qemu-kvm.
device should be defined in "BlockdevAction".
I can remove that line if it's confused.

{ 'command': 'transaction',
  'data': { 'actions': [ 'BlockdevAction' ] } }

#  List of:
#  @BlockdevAction: information needed for the device snapshot
#
# Returns: nothing on success
#          If @device is not a valid block device, DeviceNotFound